.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. 
+2.
+3.
+4.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Environment (please complete the following information):**
+ - **Collection version** (e.g. `helm ls -n sumologic`):
+ - **Kubernetes version** (e.g. `kubectl version`):
+ - **OS** (e.g. from /etc/os-release):
+ - **Kernel** (e.g. `uname -a`):
+ - **Others** (e.g. logs, cloud provider):
+
+**Anything else do we need to know** 
+Add any other context about the problem here, e.g. issue happens only occasionally.
```

Comment:
Does the final result ( in terms of formatting) is as expected?
https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/da244555b15a4eb6773e7dbb65f0512877d84860/.github/ISSUE_TEMPLATE/bug_report.md

Reply:
it doesn't matter as I think about that

Reply:
I expect that final result will be without the table.